### ISSUE TITLE
[Future] Rewrite the backend to new Nexus Prisma plugin

### ISSUE THREAD

**Issue by sMteX** [panter/ra-data-prisma#61]
## This is a future issue

Yesterday I stumbled upon https://github.com/graphql-nexus/nexus-plugin-prisma/issues/1039 . TL;DR of this is that `nexus-plugin-prisma` is no longer continued nor supported and there is ongoing project for rewriting the plugin to make it more maintainable (AFAIK on https://github.com/prisma/nexus-prisma/ )

It will probably take a while before things are completely unusable, but after this is done, the backend will need to change somehow accordingly.

Just wanted to have this somewhat documented when its time comes.

**Comment by macrozone** [comment]
yes, i am also aware of that. Should not be too hard to adopt i guess (or i hope), but in the meantime i hope they push updates to nexus-plugin-prisma until the new version is production ready (and has the features we need)
